src/entities/events/ui/event-card/event-card.module.scss

```diff
@@ -89,15 +89,14 @@
         overflow: hidden;
         display: -webkit-box;
         -webkit-box-orient: vertical;
+        -webkit-line-clamp: 2;
```

Comment:
what about to integrate `autoprefixer` package?

Reply:
Maybe we should discuss this on the daily?